### PR TITLE
Dev ei 935 2

### PR DESCRIPTION
**Pull request by Don-Olu**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+    rev: v4.4.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     # -   id: check-yaml
     #     args: [--allow-multiple-documents]
     -   id: check-added-large-files
 - repo: https://github.com/hmcts/cpp-hooks-pre-commit-terraform
-  rev: main
+  rev: eb20c9cc3f11b1f60698fa3ce9ed32f19c603ca2
```
Comment (akozichev) [inline .pre-commit-config.yaml:11]: Please change it back to  main

**Review by akozichev — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (Don-Olu) [reply .pre-commit-config.yaml]: fixed

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -13,10 +13,10 @@ repos:
     - id: terraform_fmt
     - id: terraform_validate
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.3.0
+  rev: v4.4.0
   hooks:
     - id: check-merge-conflict
 - repo: https://github.com/hmcts/cpp-hooks-gitleaks
-  rev: main
+  rev: 8674df8b3bc84211bf951057daf23b081f5ca56a
```
Comment (akozichev) [inline .pre-commit-config.yaml:20]: why are you changing revision to commitid?

File: variables.tf [hunk 3]
```diff
@@ -536,12 +541,42 @@ variable "gatekeeper_config" {
...
+
+variable "pgadmin_admin_password" {
+  type        = string
+  description = "enable pgadmin interface within cluster"
+}
+
+variable "pgadmin_oauth2_tenantid" {
+  type        = string
+  description = "pgadminoauth2_tenantId from app reg"
+  default     = "e2995d11-9947-4e78-9de6-d44e0603518e"
```
Comment (akozichev) [inline variables.tf:564]: should not go to public repo

File: variables.tf [hunk 4]
```diff
@@ -536,12 +541,42 @@ variable "gatekeeper_config" {
...
+}
+variable "pgadmin_oauth2_clientid" {
+  type        = string
+  description = "pgadmin_oauth2_clientid"
+  default     = "8e94d4eb-cf52-4eba-aeb4-4104b5cc3718"
```
Comment (akozichev) [inline variables.tf:569]: should not go to public repo

**Review by akozichev — CHANGES_REQUESTED** [review]

**Pull request merged**